cluster-autoscaler/core/scale_down.go

```diff
@@ -124,9 +128,25 @@ func (sd *ScaleDown) UpdateUnneededNodes(
 	nodeNameToNodeInfo := schedulercache.CreateNodeNameToInfoMap(pods, nodes)
 	utilizationMap := make(map[string]float64)
 
+	// Filter out nodes that were recently checked
+	nodesToCheck := make([]*apiv1.Node, 0)
+	for _, node := range managedNodes {
+		if unremovableTimestamp, found := sd.unremovableNodes[node.Name]; found {
+			if unremovableTimestamp.After(timestamp) {
+				continue
+			}
+			delete(sd.unremovableNodes, node.Name)
+		}
+		nodesToCheck = append(nodesToCheck, node)
+	}
+	skipped := len(managedNodes) - len(nodesToCheck)
+	if skipped > 0 {
+		glog.V(1).Infof("Ignoring %v recently checked nodes in scale-down calculation", skipped)
```

Comment:
This comment is misleading. What about something like: 
scale-down calculations: ignoring %v nodes, that were unremovable in the last %s minutes

Reply:
done